[OTEL-2371] update a warning in agent otlp ingestion (#28490)
DataDog/documentation: content/en/opentelemetry/setup/otlp_ingest_in_the_agent.md

   - Set `DD_OTLP_CONFIG_LOGS_ENABLED` to true.

<div class="alert alert-warning">

> <strong>Known Issue</strong>: Starting from Agent versions 7.61.0, the OTLP ingestion pipelines may fail to start in Docker environments showing the error: <code>Error running the OTLP ingest pipeline: failed to register process metrics: process does not exist</code>.<br>

Reviewer: ```suggestion
<strong>Known Issue</strong>: Starting with Agent version 7.61.0, OTLP ingestion pipelines may fail to start in Docker environments, displaying the error: <code>Error running the OTLP ingest pipeline: failed to register process metrics: process does not exist</code>.<br><br>
```

Reviewer: The additional `<br>` tags should help with the spacing. I'd recommend adding one more `<br>` tag after step 3. (GitHub won't let me add it as a suggestion.)

> If you are using an affected version, you can use one of these workarounds:<br>
> 1. Set the environment variable <code>HOST_PROC</code> to <code>/proc</code> in your Agent Docker container.<br>
> 2. Remove <code>/proc/:/host/proc/:ro</code> from <code>volumes</code> in your Agent Docker container.<br>

Reviewer: ```suggestion
If you are using an affected version, you can use one of these workarounds:<br><br>
1. Set the environment variable <code>HOST_PROC</code> to <code>/proc</code> in your Agent Docker container.<br>
2. Remove <code>/proc/:/host/proc/:ro</code> from <code>volumes</code> in your Agent Docker container.<br>
```
